src/youtube/players/YoutubeDesktopArticleVideoPlayer.tsx

```diff
@@ -59,34 +58,31 @@ const DesktopArticleVideoWrapper = styled.div<DesktopArticleVideoWrapperProps>`
 			  `}
 `;
 
-// const TopBar = styled.div`
-// 	width: 100%;
-// 	position: relative;
-// `;
+const TopBar = styled.div`
+	width: 100%;
+	position: relative;
+`;
 
-const DesktopArticleVideoPlayer: React.FC = () => {
+const YoutubeDesktopArticleVideoPlayer: React.FC<YoutubeArticleVideoPlayerProps> = ({ youtubeTakeoverDetails }) => {
 	const placeholderRef = useRef<HTMLDivElement>(null);
 	const onScreen = useOnScreen(placeholderRef, '0px', 0.1);
-	// const [dismissed, setDismissed] = useState(false);
-	// const isScrollPlayer = !(dismissed || onScreen);
-	const isScrollPlayer = !onScreen;
+	const [dismissed, setDismissed] = useState(false);
+	const isScrollPlayer = !(dismissed || onScreen);
+	// const isScrollPlayer = !onScreen;
```

Comment:
Still need this or good to remove it?

Reply:
Fixed in 46fc2fabe41513a843613e8bf199f6abf9bb8607